File: 0412-fizz-buzz/0412-fizz-buzz.py
class Solution:
    def fizzBuzz(self, n: int) -> List[str]:
        
        return [("FizzBuzz" if (x+1) % 3 == 0 and (x+1) % 5 == 0 else "Fizz" if (x+1) % 3 == 0 else "Buzz" if (x+1)%5 == 0 else str(x+1)) for x in range(n) ]

File: 0412-fizz-buzz/0412-fizz-buzz.js
/**
 * @param {number} n
 * @return {string[]}
 */
var fizzBuzz = function(n) {
    return Array(n).fill(null).map((_,i) => (i+1) % 3 == 0 && ( (i+1) % 5) == 0 ? "FizzBuzz" : (i+1) % 3 === 0 ? "Fizz" :( i+1) % 5 === 0 ? "Buzz" : (i+1).toString());
};